src/pymachine/utils.py

```diff
@@ -138,13 +173,8 @@ def to_dot(self):
         for node, n_data in self.G.nodes(data=True):
             d_node = Machine.d_clean(node)
             printname = Machine.d_clean('_'.join(d_node.split('_')[:-1]))
-            if not n_data['expanded']:
```

Comment:
please also these 7 lines which use n_data, if present. The online demo relies on this code.